src/services/bookmark.service.js

```diff
@@ -0,0 +1,24 @@
+import db from '../database/models/index';
+
+const BookmarkService = {
+  async getBookmarkedAccommodation(UserId) {
+    const result = await db.Accommodations.findAll({
+      include: [{ model: db.Bookmarks, where: { UserId }, attributes: [] }],
+    });
+    return result;
+  },
+  async bookmark(data) {
+    const isBookmarked = await db.Bookmarks.findOne({ where: data });
+    let result;
+    if (!isBookmarked) {
+      await db.Bookmarks.create(data);
+      result = 'added';
+    } else {
+      await db.Bookmarks.destroy({ where: data });
+      result = 'deleted';
```

Comment:
`post` is for creating and `delete` is for deleting!
Imagine a user made a `post` request and receive a message saying `Bookmark deleted successfully`! it won't make sense!
please try to separate these functionalities bruh!

Reply:
okey